### PR TITLE
Updated the README.md file with an optional field to add deprecated d…

### PR DESCRIPTION
**Pull request by pubudup**
- Updated the README.md file with an optional field to add deprecated details when applicable.
- Also, committed a script that restarts WSO2 products at pre-defined time intervals and run in background.

### CODE REVIEW

**Comment by nu1silva** [comment]
Better if we can make this into a cronjob
- Better control over when to start/stop
- Can manage efficiently

**Comment by pubudup** [comment]
Agreed to send a separate script to run as a cron job.

File: tools/scripts/bash_scripts/Script_to_Restart_WSO2_Server/restart.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+#finds wso2 product prodess in provided location and kills it. Then restarts the server after a defined time period. Continues to do this until the script is stopped.
+DOWNTIME=1m
+UPTIME=1m
+while [ true ]
+do
+PID=`ps -ef | grep /home/pubudup/Documents/WSO2_Products/MB/320/M2/25Aug/n1/wso2mb-3.2.0-M2/bin | awk '{ print $2 }'`
```
Comment (dakshika) [inline tools/scripts/bash_scripts/Script_to_Restart_WSO2_Server/restart.sh:7]: Path should be configurable. Do not commit local machine paths

File: tools/scripts/bash_scripts/Script_to_Restart_WSO2_Server/restart.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+kill -9 $PID
+echo WSO2 server killed
+sleep $DOWNTIME
+cd /home/pubudup/Documents/WSO2_Products/MB/320/M2/25Aug/n1/wso2mb-3.2.0-M2/bin
```
Comment (dakshika) [inline tools/scripts/bash_scripts/Script_to_Restart_WSO2_Server/restart.sh:11]: Path should be configurable. Do not commit local machine paths

**Review by dakshika — COMMENTED** [review]
Path should be configurable. Do not commit local machine paths

**Comment by pubudup** [comment]
@dakshika @ushanib : New pull request sent with requested changes. Since this requires to be run on the same server as WSO2 product we need to provide the absolute path. I have parameterized the absolute path and updated this in readme file with instructions now. 

Pull request: https://github.com/wso2/wso2-qa-artifacts/pull/127

**Pull request merged**